### PR TITLE
twoliter: bump to bottlerocket-core-kit v3.3.2

### PR DESCRIPTION
**Pull request by piyush-jena**
**Description of changes:**

Update to the new core kit release.

### CODE REVIEW

**Review by Sparksssj — APPROVED** [review]

**Review by yeazelm — APPROVED** [review]
LGTM!

**Review by Sparksssj — APPROVED** [review]
LGTM if the removing spaces not a big deal

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+# v1.27.1 (2024-11-16)
+
+## Release Highlights
+* Add kernel-5.15 patch to fix IPv6 typo ([bottlerocket-core-kit#266](https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/266))
```
Comment (yeazelm) [inline CHANGELOG.md]: nit:
```suggestion
* Add patch for kernel-5.15 to fix issues when using IPv6 ([bottlerocket-core-kit#266](https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/266))
```

Comment (piyush-jena) [reply CHANGELOG.md]: fixed

**Review by yeazelm — APPROVED** [review]

**Review by KCSesh — APPROVED** [review]

**Pull request merged**